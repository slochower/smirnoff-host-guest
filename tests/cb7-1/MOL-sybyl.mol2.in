#!/usr/bin/env bash
source $AMBERHOME/amber.sh

        antechamber -i ./tests/cb7-1/MOL.mol2 -fi mol2 -o ./tests/cb7-1/MOL-sybyl.mol2 -fo mol2 -at sybyl
        