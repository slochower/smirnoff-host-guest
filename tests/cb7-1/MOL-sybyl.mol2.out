
Welcome to antechamber 17.3: molecular input file processor.

acdoctor mode is on: check and diagnosis problems in the input file.
-- Check Format for mol2 File --
   Status: pass
-- Check Unusual Elements --
   Status: pass
-- Check Open Valences --
   Status: pass
-- Check Geometry --
      for those bonded   
      for those not bonded   
   Status: pass
-- Check Weird Bonds --
   Status: pass
-- Check Number of Units --
   Status: pass
acdoctor mode has completed checking the input file.


